img_up/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+import uuid
+from django.contrib.auth.models import User
+
+
+# Create your models here.
+class Epilepsy(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    admin_name = models.CharField(max_length=30)
+    state = models.CharField(max_length=30)
+    numOfTotalCases = models.PositiveSmallIntegerField()
+    numOfChildCases = models.PositiveSmallIntegerField()
+    numOfAdultCases = models.PositiveSmallIntegerField()
```

Comment:
Not sure why you're getting that integer error. I'll have to observe the behavior myself. I'll get in touch after lecture/lunch